gravitee-apim-console-webui/src/management/api/apis.route.ts

```diff
@@ -825,29 +825,29 @@ export const states: Ng2StateDeclaration[] = [
     },
   },
   {
-    name: 'management.apis.notifications',
-    component: NotificationsListComponent,
-    url: '/notifications',
+    name: 'management.apis.notification-settings',
+    component: ApiNotificationSettingsListComponent,
+    url: '/notification-settings',
     data: {
       apiPermissions: {
         only: ['api-notification-r'],
       },
       docs: {
-        page: 'management-api-notifications',
+        page: 'management-api-notification',
```

Comment:
```suggestion
        page: 'management-api-notifications',
```

These are doc page keys and we're not changing them for the moment. This key does not exist

Reply:
Done